internal/process/defunct_processes.go

```diff
@@ -80,11 +80,11 @@ func status(pid string) (*Stat, error) {
 	}
 
 	stat := &Stat{
-		// The executable filename is field 2.
-		Fname: parts[1],
+		// The command name is field 2.
+		Comm: parts[1],
 
 		// The state is field 3, which is the first two fields and a space after.
 		State: string(data[i+2]),
 	}
 	return stat, nil
-}
+}
```

Comment:
nit: no newline at EOF

Reply:
Thank you for the catch! Added a newline :)